battleship.py

```diff
@@ -203,6 +239,15 @@ def placeShip(data):
 Returns: None
 '''
 def clickUserBoard(data, row, col):
+    if data["userships"]==5:
+       print("You can start the game")
+       return
+    if [row,col] == data["tempship"]:
+       return 

```

Comment:
I think it should be 'in' not '=='.  Check once.

Reply:
Also, resolve conflicts by deleting pycache and then clicking on the resolve conflicts button for battleship.py